beacon-chain/rpc/eth/validator/handlers_block.go

```diff
@@ -182,13 +191,30 @@ func (s *Server) ProduceBlockV3(w http.ResponseWriter, r *http.Request) {
 	}
 
 	s.produceBlockV3(ctx, w, r, &eth.BlockRequest{
-		Slot:         primitives.Slot(slot),
-		RandaoReveal: randaoReveal,
-		Graffiti:     graffiti,
-		SkipMevBoost: false,
+		Slot:               primitives.Slot(slot),
+		RandaoReveal:       randaoReveal,
+		Graffiti:           graffiti,
+		SkipMevBoost:       false,
+		BuilderBoostFactor: &wrapperspb.UInt64Value{Value: bbFactor},
 	}, any)
 }
 
+func processBuilderBoostFactor(raw string) (uint64, error) {
+	trimmed := strings.ReplaceAll(raw, " ", "")
+	switch trimmed {
+	case "": // default to 100 if it's not provided
+		return 100, nil
+	case "2**64-1":
```

Comment:
Lol I thought that the user would have to specify explicitly `18446744073709551615`